### ISSUE TITLE
Cannot set label vertical padding on iOS

### ISSUE THREAD

**Issue by erkanarslan** [NativeScript/NativeScript#5194]
### Did you verify this is a real problem by searching [Stack Overflow]
Yes

### Tell us about the problem
I cannot set vertical padding of label elements on iOS. Padding values are applied like margin values. New space is added but background-color is not applied to that area. Because of that it looks same as applying margin.

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- CLI: 3.3.1
- Cross-platform modules: 3.3.0
- Runtime(s): 3.3.0
### Please tell us how to recreate the issue in as much detail as possible.
Create a label element.
Set its background color.
Set its padding to something large.
Vertiacal padding is not applied.

**Comment by tsonevn** [comment]
Hi @erkanarslan,
Thank you for contacting us.
I tested your scenario with the latest version of NativeScript 3.4, which was released yesterday and was unable to recreate the similar issue with the vertical padding on iOS. For your convenience, I am attaching sample project and Gif file, which demonstrates how it is working.
Also please review the project and make the needed changes, which will allow us to recreate this behavior.
[Archive.zip](https://github.com/NativeScript/NativeScript/files/1578157/Archive.zip)
![screencast 2017-12-21 at 8 27 57 am](https://user-images.githubusercontent.com/17448734/34243501-c71d1756-e629-11e7-96b8-abaa588acd6d.gif)

**Comment by erkanarslan** [comment]
@tsonevn I noticed that padding is set incorrectly if label is inside flexbox-layout. Here are my xml, css codes and images from app:

The element with padding is the one that has `task-state` class.
````xml
<stack-layout class="task priority-border priority-{{task.priority}}">
	<label [text]="task.name" class="task-name"></label>
	<flexbox-layout class="task-details">
		<flexbox-layout class="flexible flex-center">
			<label [text]="task.state.name" class="task-state" [ngStyle]="{'background-color' : task.state.color, color : task.state.textColor}"></label>
			<label [text]="task | readableInterval" class="date flexible {{task.completedAt ? '' : (task.deadline | isOverdue) ? 'text-danger' : 'text-warning'}}"></label>
		</flexbox-layout>
		<flexbox-layout>
			<image *ngFor="let user of task.assignees" [src]="user.profile" class="user-profile"></image>
		</flexbox-layout>
	</flexbox-layout>
</stack-layout>
````

````css
.task-state {
	border-radius: 4;
	color: white;
	padding: 2 6;
	margin-right: 8;
}
````

Look at the green element. In the first image below, padding is set as shown in the css code above. In the second image, padding is set to `6 6`. Even though padding value is set to a greater value in the second one, its vertical padding is 0 or something very small.

![img_0031](https://user-images.githubusercontent.com/11949947/34245809-bfabde5e-e63c-11e7-83b1-63d2b3aacfa9.PNG)
![img_0032](https://user-images.githubusercontent.com/11949947/34245811-c11fcfca-e63c-11e7-8a88-b435c8e1e6b1.PNG)

**Comment by tsonevn** [comment]
Hi @erkanarslan,
I reused some part of the above-provided code samples, however, was unable to recreate the exact same behavior like the described one. For your convenience, I attached NS Angular sample project. Please review it again and make the needed changes, which will allow us to recreate the same behavior.
[Archive.zip](https://github.com/NativeScript/NativeScript/files/1597825/Archive.zip)

**Comment by erkanarslan** [comment]
It looks like this is fixed in version 3.4.0.
Thanks.
